src/main/java/com/connexta/search/common/configs/SolrConfiguration.java

```diff
@@ -22,8 +22,10 @@
   public static final String SOLR_COLLECTION = "searchTerms";
   public static final String CONTENTS_ATTRIBUTE_NAME = "contents";
   public static final String ID_ATTRIBUTE_NAME = "id";
+  public static final String MEDIA_TYPE_ATTRIBUTE_NAME = "media.type";
```

Comment:
❗️Add test(s) for querying on this term

Reply:
It makes sense to have a test that ensures every supported attribute is indexed properly and can also be queried. I'll add that.